sdk/src/integrationTest/java/TopicMessageSubmitIntegrationTest.java

```diff
@@ -28,37 +28,37 @@ void canSubmitATopicMessage() {
 
             @Var var info = new TopicInfoQuery()
                 .setTopicId(topicId)
-                .setNodeAccountIds(testEnv.nodeAccountIdsForChunked)
+                //.setNodeAccountIds(testEnv.nodeAccountIdsForChunked)
```

Comment:
Note to self: search and replace for `//.setNodeAccountIds(testEnv.nodeAccountIdsForChunked)`